<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferPage</class>
 <widget class="QWidget" name="TransferPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>482</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(192, 196, 212);</string>
  </property>
  <widget class="QLabel" name="addressLabel2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>来自地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>110</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="changeAccountLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>来自</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="memoLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>备注</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sendtoLabel">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发往</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>220</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>400</y>
     <width>145</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgb(24,196,252), stop:1 rgb(24,128,252)); border-radius:5px;color: rgb(255, 255, 255);}
QToolButton:hover{background-color:qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgb(42,215,254), stop:1 rgb(28,151,255));}
QToolButton:disabled{background-color:qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgb(83,90,109), stop:1 rgb(70,76,93));color: rgb(40,45,57);}</string>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="amountAssetLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>UB</string>
   </property>
  </widget>
  <widget class="QLabel" name="feeLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>手续费</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="feeLineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0.0001</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="unitLabel2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>UB</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>375</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>余额不足</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel4">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>190</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>无效地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="delegateLabel2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendtoLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>长度超过40个字节</string>
   </property>
  </widget>
  <widget class="QComboBox" name="assetComboBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>72</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(177,177,177);</string>
   </property>
   <property name="text">
    <string>如需修改手续费请到设置窗口</string>
   </property>
  </widget>
  <widget class="QLabel" name="callContractFeeLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="callContractFeeLabel2">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>280</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>UB</string>
   </property>
  </widget>
  <widget class="QLabel" name="callContractFeeLabel3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>280</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预计合约调用费用</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="memoTextEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>360</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{background-color: rgb(32,36,54);color: rgb(192, 196, 212);border:1px solid rgb(28, 32, 46);padding: 0px 10px 0px 6px;}
QTextEdit:focus{border:1px solid rgb(93,107,144);}

</string>
   </property>
  </widget>
  <widget class="QComboBox" name="contactComboBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>72</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <zorder>memoTextEdit</zorder>
  <zorder>addressLabel2</zorder>
  <zorder>addressLabel</zorder>
  <zorder>changeAccountLabel</zorder>
  <zorder>memoLabel</zorder>
  <zorder>sendtoLabel</zorder>
  <zorder>amountLabel</zorder>
  <zorder>sendBtn</zorder>
  <zorder>amountLineEdit</zorder>
  <zorder>feeLabel</zorder>
  <zorder>feeLineEdit</zorder>
  <zorder>unitLabel2</zorder>
  <zorder>tipLabel3</zorder>
  <zorder>tipLabel4</zorder>
  <zorder>delegateLabel2</zorder>
  <zorder>accountComboBox</zorder>
  <zorder>sendtoLineEdit</zorder>
  <zorder>tipLabel6</zorder>
  <zorder>assetComboBox</zorder>
  <zorder>amountLabel_2</zorder>
  <zorder>callContractFeeLabel</zorder>
  <zorder>callContractFeeLabel2</zorder>
  <zorder>callContractFeeLabel3</zorder>
  <zorder>amountAssetLabel</zorder>
  <zorder>contactComboBox</zorder>
 </widget>
 <tabstops>
  <tabstop>amountLineEdit</tabstop>
  <tabstop>feeLineEdit</tabstop>
  <tabstop>sendBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
